### PR TITLE
Dynamic grouping now possible.

### PR DESCRIPTION
**Pull request by blattmann**
Constant value(s) in "index.html" should be provided by the back end in the future.

### CODE REVIEW

**Comment by fgravin** [comment]
Perfect thanks

**Comment by sbrunner** [comment]
Thanks, but Travis isn't happy :-(

**Comment by fgravin** [comment]
Travis is failing.
Did you do a `make check` ?

You use the `gmfSearchGroups` value but haven't defined it in desktop html file ....

File: contribs/gmf/apps/mobile/index.html [hunk 1]
```diff
@@ -119,6 +119,7 @@
          module.constant('fulltextsearchUrl', 'https://geomapfish-demo.camptocamp.net/2.0/wsgi/fulltextsearch');
          module.constant('gmfWmsUrl', 'https://geomapfish-demo.camptocamp.net/2.0/wsgi/mapserv_proxy');
          module.constant('gmfTreeUrl', 'https://geomapfish-demo.camptocamp.net/2.0/wsgi/themes?version=2&background=background');
+         module.constant('gmfSearchGroups', ['osm','district']);
```
Comment (sbrunner) [inline contribs/gmf/apps/mobile/index.html]: Should also be added in the desktop layout :-)

**Comment by blattmann** [comment]
Sorry. I added the missing value to the desktop layout.

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**